src/arsenic/session.py

```diff
@@ -315,6 +315,10 @@ async def switch_to_window(self, handle):
             url="/window", method="POST", data={"handle": handle, "name": handle}
         )
 
+    async def new_window(self, window_type: WindowType = None):
+        data = {"type": window_type} if window_type is not None else None
```

Comment:
The WebDriver spec refers to `parameters` argument... but then again it does so for other operations, and this file doesn't seem do that, so I guess it's OK? 🤷🏿

Reply:
I think that's okay, but we can also ask someone else about this.